frontend/src/app/shared/components/op-content-loader/op-content-loader.component.html

```diff
@@ -1,5 +1,5 @@
 <content-loader
-  [backgroundColor]="'var(--bgColor-muted)'"
+  [backgroundColor]="'var(--skeletonLoader-bgColor)'"
```

Comment:
That change does not really make difference. It even looks like it is darker than before, so even worse visible

**Before**
<img width="1956" alt="Bildschirmfoto 2024-07-17 um 12 29 11" src="https://github.com/user-attachments/assets/21190e27-2138-4a00-a340-45d63842ae80">


**After**
<img width="1950" alt="Bildschirmfoto 2024-07-17 um 12 30 38" src="https://github.com/user-attachments/assets/6324f777-b145-46de-82c5-231c7917ca2a">